descriptors/ModuleDescriptor-template.json

```diff
@@ -580,6 +631,7 @@
         "organizations-storage.contacts.all",
         "organizations-storage.emails.all",
         "organizations-storage.interfaces.all",
+        "organizations-storage.interfaces.credentials.all",
```

Comment:
I'm not sure if this is a good idea to add this permission to "all".as the whole idea was to restrict the permission as and when required

Reply:
Anyway we can leave it here as it makes sense to combine with interfaces endpoint